pontoon/base/static/js/translate.js

```diff
@@ -231,6 +231,23 @@ var Pontoon = (function (my) {
     },
 
 
+    getApproveButtonTitle: function(translation) {
+      var approveButtonTitle = '';
+
+      /* Moved the logic from ternary operators to make this a little bit easier to read. */
+      if (translation.approved && translation.approved_user) {
+        return 'Approved by ' + translation.approved_user;
+      } else {
+        if (translation.unapproved_user) {
+          return 'Unapproved by ' + translation.unapproved_user;
+        } else {
+          return 'This translation requires verification & approval.';
```

Comment:
We should also add this text if we don't have user email and display `Imported` instead - see line **286**. But only if translation is not approved. I'd change the text to: "This translation has not been reviewed yet.".

Actually, we can skip this case and leave it for the next related bug, that will introduce the **unreviewed** state, for which we'll have to change the UI anyways:
https://bugzilla.mozilla.org/show_bug.cgi?id=1229335#c5